packages/blockchain-wallet-v4-frontend/src/components/Terms/index.tsx

```diff
@@ -80,7 +80,7 @@ const Terms = (props) => {
             data-e2e='blockchainTermsLink'
           >
             <FormattedMessage
-              id='scenes.register.registerform.blockchain.default.terms'
```

Comment:
why did you change this line?

Reply:
because it seems there is a bug in production ? take a look to the screenshot
<img width="384" alt="image" src="https://user-images.githubusercontent.com/97241313/162591640-739e7111-3204-49e1-9473-fe6a98073b04.png">